site/src/components/DeploySettingsLayout/Option.tsx

```diff
@@ -38,7 +39,7 @@ export const OptionDescription: FC<PropsWithChildren> = (props) => {
 };
 
 interface OptionValueProps {
-  children?: boolean | number | string | string[];
+  children?: boolean | number | string | string[] | Record<string, boolean>;
```

Comment:
That's a good idea and I would love to massage the data into, say, an array in the parent; however, I'm actually using an object here purposely to differentiate from how [we normally display lists](https://github.com/coder/coder/blob/aa65ff027d0655cc449327541842f91dcc9a59be/site/src/components/DeploySettingsLayout/Option.tsx#L114). 
In addition, if we use `PropsWithChildren`, we can no longer check [the length of the child](https://github.com/coder/coder/blob/aa65ff027d0655cc449327541842f91dcc9a59be/site/src/components/DeploySettingsLayout/Option.tsx#L77) without using the `Children` API, which I was hoping to avoid.
This seemed like the simplest path forward but open to suggestions!

Reply:
Sure, maybe `optionValues` or something. I will think it over and push up a change!